rest/route/admin_events.go

```diff
@@ -44,16 +43,9 @@ func (h *adminEventsGet) Parse(ctx context.Context, r *http.Request) error {
 		}
 	}
 
-	if l, ok := vals["limit"]; ok && len(l) > 0 {
-		h.Limit, err = strconv.Atoi(l[0])
-		if err != nil {
-			return errors.Wrap(err, "problem parsing limit")
-		}
-	}
-
-	// if user asks for a limit of 0 we should override.
-	if h.Limit == 0 {
-		h.Limit = 10
+	h.Limit, err = getLimit(vals)
+	if err != nil {
+		return errors.WithStack(err)
```

Comment:
In general I don't think we should return stacks with errors in rest routes

Reply:
I think it makes debugging tests easier, and allows us to log errors with higher fidelity. The `getLimit` method returns an error, so I'm just propogating the chain in the annotation.

Additionally any pkg/errors method annotates errors with stacks and messages, as needed. 

The consumers in the gimlet framework (and in legacy restv2) unwrap errors, and never return stacktraces to users, so I think its safe.